### PR TITLE
add a few programs

### PR DESCRIPTION
**Pull request by SSilence**

### CODE REVIEW

**Comment by nodiscc** [comment]
:+1: Thank you

File: README.md [hunk 1]
```diff
@@ -429,6 +431,7 @@ CMS are a practical way to setup a website with many features. CMS often come wi
   * [LimeSurvey](https://www.limesurvey.org) - Feature-rich Open Source web based polling software. Supports extensive survey logic. - ([Demo](https://demo.limesurvey.org))
   * [MaterialPoll](https://github.com/nearbycoder/materialpoll) - an opensource strawpoll.me clone for creating random polls. - ([Source code](https://github.com/nearbycoder/materialpoll), [Demo](http://materialpoll.tk))
   * [OpenSondage](https://github.com/leblanc-simon/OpenSondage) - an online service for planning an appointment or make a decision quickly and easily: Make a poll, Define dates or subjects to choose, Send the poll link to your friends or colleagues, Discuss and make a decision - ([Demo](http://framadate.org/), [Example](https://framadate.org/aqg259dth55iuhwm))
+  * [scotty](http://www.scotty-transporter.org/) - Scotty is a free opensource proxy software for bypassing filter and censorship systems. - ([Source Code](https://github.com/SSilence/scotty)) `GPLv3` `Java` `PHP`
```
Comment (nodiscc) [inline README.md]: Would be better with `Java/PHP` (I'm just nitpicking)

File: README.md [hunk 2]
```diff
@@ -375,6 +376,7 @@ CMS are a practical way to setup a website with many features. CMS often come wi
   * [Jotter](https://github.com/yosko/jotter) - Jotter is a lightweight, no database, powerful web notebook that lets you create and manage notes online safely, quickly & easily. ([Demo](http://tools.yosko.net/demos/jotter/) - `demo/demo`)
   * [Laverna](https://laverna.cc/) - Laverna is a JavaScript note taking application with Markdown editor and encryption support. Consider it like open source alternative to Evernote ([Source code](https://github.com/Laverna/laverna), [Demo](https://laverna.cc/index.html#notes))
   * [minimalist-web-notepad](https://github.com/pereorga/minimalist-web-notepad)- a minimalist http://notepad.cc clone `Apache` `PHP`
+  * [notes](https://github.com/SSilence/notes) - Simple script for personal notes with a single page with a WYSIWYG Editor `Apache` `PHP` `GPLv3`
```
Comment (nodiscc) [inline README.md]: I think license is just GPLv3

**Comment by Kickball** [comment]
Just waiting on the minor changes nodiscc mentioned before we merge.

**Comment by nodiscc** [comment]
Thanks @SSilence , merged.

**Pull request merged**